deps/wazuh_testing/wazuh_testing/tools/remoted_sim.py

```diff
@@ -607,6 +608,9 @@ def update_keys(self):
         """
         Update keys table with keys read from client.keys
         """
+        if not os.path.exists(self.client_keys_path):
+            with open(self.client_keys_path, 'w+') as f:
+                f.write("100 ubuntu-agent any TopSecret")
```

Comment:
Add a new line to visually separate this `if` block from the context manager from below.

Reply:
https://github.com/wazuh/wazuh-qa/pull/1319/commits/21084af927121694ca723501b2bcc3603c0d1c13